src/containers/PageBuilder/Primitives/SearchCTA/FilterCategories/FilterCategories.js

```diff
@@ -0,0 +1,111 @@
+import React, { useState } from 'react';
+import { Field } from 'react-final-form';
+import classNames from 'classnames';
+import { FormattedMessage } from '../../../../../util/reactIntl';
+
+import { OutsideClickHandler } from '../../../../../components';
+
+import css from './FilterCategories.module.css';
+
+const CategoryDropdown = ({ input, className, rootClassName, categories }) => {
+  const [isOpen, setIsOpen] = useState(false);
+  const [hasSelected, setHasSelected] = useState(false);
+
+  const toggleDropdown = () => setIsOpen(!isOpen);
+
+  const handleOptionClick = optionId => {
+    // we want to submit an empty value to the form if the "All categories" value is selected
+    const value = optionId === 'all-categories' ? '' : optionId;
+    input.onChange(value);
+    setHasSelected(true);
+    setIsOpen(false);
+  };
+
+  // Identify the selected category
+  const selectedCategory = categories.find(category => category.id === input.value);
+
+  // Set the dropdown label: we store a hasSelected value in state to check if the user
+  // has interacted with the form yet, i.e. selected a value. If not, we show a placeholder value
+  const labelText = selectedCategory ? (
+    selectedCategory.name
+  ) : hasSelected && input.value === '' ? (
+    <FormattedMessage id="CategoryFilter.selectAll" />
+  ) : (
+    <FormattedMessage id="CategoryFilter.placeholder" />
+  );
+
+  const rootClass = rootClassName || css.root;
+  const classes = classNames(rootClass, className);
+
+  return (
+    <OutsideClickHandler className={classes} onOutsideClick={() => setIsOpen(false)}>
+      <div className={css.dropdownContainer}>
+        <div
+          className={classNames(
+            css.toggleButton,
+            // If no selection has been made, no category is selected, and dropdown is closed, apply 'unselected' style
+            // else, if dropdown is open but no selection has been made, apply placeholderOpened style
+            !hasSelected && !selectedCategory && !isOpen && css.unselected,
+            isOpen && !hasSelected && css.placeholderOpened
```

Comment:
```suggestion
            css.toggleButton,
            {
              // If no selection has been made, no category is selected, and dropdown is closed, apply 'unselected' style
              // else, if dropdown is open but no selection has been made, apply placeholderOpened style
              [css.unselected]: !hasSelected && !selectedCategory && !isOpen,
              [css.placeholderOpened]: isOpen && !hasSelected,
            }
```

Do I understand this right: 
When filter is open and there's no selected category nor "all values", the grey700 color is used => aka filter shows as active?

(I find this slightly counter-intuitive when compared to <input> behaviour: if there's no value, placeholder text is shown as grey => i.e. whenever placeholder text is visible, it's grey)

Reply:
@Gnito placeholder should be shown grey700 when the dropdown is active:

![image](https://github.com/user-attachments/assets/185ce497-5272-49bd-aee1-10d816a3ff1d)